utils/swift_build_sdk_interfaces.py

```diff
@@ -54,9 +54,8 @@ def create_parser():
     parser.add_argument('-F', dest='framework_dirs', metavar='DIR',
                         action='append', default=[],
                         help='Add additional framework search paths')
-    parser.add_argument('-Fsystem', '-iframework',
-                        dest='system_framework_dirs', metavar='DIR',
-                        action='append', default=[],
+    parser.add_argument('-Fsystem', dest='system_framework_dirs',
```

Comment:
This is the input argument to `swift_buil_sdk_interfaces.py` which I think is deprecated already. @artemcm @nkcsgexi Is that correct?

Seems irrelevant to your fix.

Reply:
I maybe have been a little too enthusiastic about removing `-iframework`. I'm still kind of tempted to drop it here, but I can put it back if you think that's better.